source/unified-test-format/unified-test-format.rst

```diff
@@ -553,6 +553,9 @@ The structure of this object is as follows:
     `Command Monitoring <../command-monitoring/command-monitoring.rst#security>`__
     spec) will be observed for this client. If false or not specified, events for
     commands containing sensitive information MUST be ignored.
+    ``runOnRequirements.auth`` MUST be false when this property is true. Test
```

Comment:
`runOnRequirements` is an array of multiple requirements, so I don't think it makes sense to say `runOnRequirements.auth`.

I'd suggest adding `.. _runOnRequirement_auth:` above the definition of:

    - ``auth``: Optional boolean...

And then just replacing this with:

    `runOnRequirement_auth`_.

You'll also need to specify that auth must be prohibited for all runOnRequirements. Doing so at either the top-level or per-test level would be sufficient, since they are evaluated independently. The goal is to avoid having a single runOnRequirement without `auth: false` be satisfied and allow the test to run on an auth-enabled cluster. 

I'd also suggest using SHOULD instead of MUST here since it's not something we'd enforce in the schema (see: [Design Rationale](https://github.com/mongodb/specifications/blob/master/source/unified-test-format/unified-test-format.rst#design-rationale)). Instead, it's guidance to help people avoid writing tests that would likely fail otherwise.

Reply:
Done.